.travis.yml

```diff
@@ -1,31 +1,33 @@
 language: python
 sudo: false
+cache: pip
+git:
+   depth: 1
 python:
 - 2.6
 - 2.7
-- 3.2
 - 3.3
 - 3.4
 - 3.5
+- 3.6
 - pypy
-- pypy3
 env:
 - secure: "EhG2uSD2m1eGxuL2HeQewJJx7MVL4WpjrxyfiUBEgsApemD1yKJPjUnLwAyd\nbPi5HJx5ySm1TTRSvj6/yP85YLYTaJHA8OabKk7p0wFW294qcrYIDGovU7NL\n3YOqZmqN+S3XOBGFCOnByxE+pcxxWW/3/I09EgeW7D6tBPh67G0="
 install:
 - echo $TRAVIS_PYTHON_VERSION
-- pip install pytest pytest-xdist pytest-capturelog memory_profiler==0.40 psutil coveralls flake8
+- pip install psutil coveralls flake8 memory_profiler
 - >
-    if [[ "$TRAVIS_PYTHON_VERSION" != "3.2" && "$TRAVIS_PYTHON_VERSION" != "pypy" && "$TRAVIS_PYTHON_VERSION" != "pypy3" ]]; then
+    if [[ "$TRAVIS_PYTHON_VERSION" != "pypy" ]]; then
       pip install 'pytest-cov>=1.8.0';
     fi
-- pip install -e .
+- pip install -e .[test] --upgrade
 script:
 - python -mwand.version --verbose
 - >
     if [[ "$TRAVIS_PYTHON_VERSION" == "3.2" || "$TRAVIS_PYTHON_VERSION" == "pypy" || "$TRAVIS_PYTHON_VERSION" == "pypy3" ]]; then
```

Comment:
If we remove Python 3.2 from Travis CI build this condition also need to be changed.